src/App.tsx

```diff
@@ -1,61 +1,97 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const todosWithUsers = todosFromServer.map(todo => {
+  const user = usersFromServer.find(userr => userr.id === todo.userId);
+
+  return {
+    ...todo,
+    user,
+  };
+});
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosWithUsers);
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/users" method="POST">
+      <form
+        action="/api/users"
+        method="POST"
+        onSubmit={(event) => {
+          event.preventDefault();
+          if (title && user) {
+            const maxIndex = Math.max(...todos.map(todo => todo.id)) + 1;
+            const newTodo = {
+              id: maxIndex,
+              title,
+              completed: false,
+              userId: +user,
+              user: usersFromServer.find(userr => userr.id === +user),
```

Comment:
what's the difference between user and userr? it's almost impossible to debug such code for other developers. Give this variable normal name